File: 프로그래머스/lv0/120904. 숫자 찾기/숫자 찾기.java
class Solution {
    public int solution(int num, int k) {
        int answer = 0;
        String[] numstr = Integer.toString(num).split("");
        for(int i=0; i<numstr.length;i++) {
        	if(Integer.toString(k).equals(numstr[i])) {
        		answer=i+1;
        		break;
        	} else {
        		answer=-1;
        	} 
        	
        }
        return answer;	
    }
}

File: 프로그래머스/lv0/120904. 숫자 찾기/숫자 찾기.cs
using System;

public class Solution {
    public int solution(int num, int k) {
        int answer = -1;
        string numString = num.ToString();
        string kString = k.ToString();
        
        if(numString.Contains(kString)) answer = numString.IndexOf(kString)+1;
        
        return answer;
    }
}